packages/playwright-test/src/docker/dockerApi.ts

```diff
@@ -0,0 +1,200 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import http from 'http';
+
+// Docker engine API.
+// See https://docs.docker.com/engine/api/v1.41/
+
+const DOCKER_API_VERSION = '1.41';
+
+export interface DockerImage {
+  imageId: string;
+  names: string[];
+}
+
+export interface DockerContainer {
```

Comment:
```suggestion
interface DockerContainer {
```

Reply:
this is used in docker.ts

Reply:
I see no import or usage! but anyways a nit.